docs/docs/core/policies.mdx

```diff
@@ -25,6 +25,9 @@ to the desired number of maximum predictions.
 
 Your project’s `config.yml` file takes a `policies` key
 which you can use to customize the policies your assistant uses.
+You can also leave this key empty. Then the
```

Comment:
Might be good to show a little code snippet for each of these "left empty" as some users might take that as not including it at all. 

"You can also leave this key empty:
```

policies: 
# intentionally left empty

```
Then the suggested...

Reply:
Nevermind I just read the actual documentation that says you can leave it out or empty -- I'd write that here